src/web/releases.rs

```diff
@@ -528,13 +564,11 @@ pub fn search_handler(req: &mut Request) -> IronResult<Response> {
                      OFFSET FLOOR(RANDOM() * 280) LIMIT 1",
                     &[]
                 ));
```

Comment:
This whole thing is pretty clearly incorrect (#789). Probably best to keep functionality changes separate from refactors though.

Reply:
I didn't change functionality any